Source/Scene/Cesium3DTileFeature.js

```diff
@@ -55,6 +53,24 @@ define([
          * @private
          */
         this.primitive = tileset;
+
+        /**
+         * All objects returned by {@link Scene#pick} have a <code>tile</code> property.
```

Comment:
Replace this comment and the one below with something else, since this is not true for non 3d-tiles things. The comment above `primitive` is still correct though.